docs/driver/compile.rst

```diff
@@ -1,5 +1,3 @@
-.. include:: ./driver/read.rst
```

Comment:
Why is this removed here?

Reply:
Leads to a weird error with the current documentation where it tries to find the read.rst at `../../driver/driver/read.rst`.

Reply:
Maybe it should just be `read.rst`.

Reply:
Doesn't work, then the `make doc` complains about:
```
../../docs/driver/compile.rst:1: CRITICAL: Problems with "include" directive path:
InputError: [Errno 2] No such file or directory: '../../docs/read.rst'.
```